data/actionsmap/yunohost.yml

```diff
@@ -925,6 +925,54 @@ monitor:
 
 
 #############################
+#         Settings          #
+#############################
+settings:
+    category_help: Manage YunoHost global settings
+    actions:
+
+        ### settings_list()
+        list:
```

Comment:
To be discussed / implemented later maybe, but it could be nice to have an option to list only differences with the default settings.